demo-app/src/app/ot/playground/page.tsx

```diff
@@ -1,12 +1,50 @@
-import { parseYaml, applyConfiguration } from '@open-truss/open-truss'
+import {
+  parseYaml,
+  applyConfiguration,
+  type UqiMetadata,
+} from '@open-truss/open-truss'
 import * as COMPONENTS from '@open-truss-components'
+import createTrinoUqiClient from '@/lib/trino-uqi-client'
+
 const configurationFunction = applyConfiguration(COMPONENTS)
 
 async function PlaygroundPage(): Promise<JSX.Element> {
+  const trinoUqiClient = await createTrinoUqiClient({
+    auth: 'trino',
+    server: 'http://trino.orb.local:8080',
+  })
+  const queryIterator = await trinoUqiClient.query(
+    'select * from  tpch.sf1.customer limit 10',
+  )
+  const rows = []
+  let metadata: UqiMetadata = {
+    columns: [],
+  }
+  for await (const { row, metadata: m } of queryIterator) {
+    rows.push(row)
+    if (metadata.columns.length === 0) {
```

Comment:
Out of curiosity what is this if block for? Just for demo purposes?

Reply:
I'm still trying to solve the problem where the client might want to keep track of metadata like column types but it only needs to capture that data once. This was a hack to do that.